datumaro/plugins/cifar_format.py

```diff
@@ -178,6 +183,10 @@ class CifarConverter(Converter):
     DEFAULT_IMAGE_EXT = '.png'
 
     def apply(self):
+        if self._extractor.media_type() and \
+                self._extractor.media_type() is not Image:
```

Comment:
```suggestion
                not issubclass(self._extractor.media_type(), Image):
```
This is more correct way of comparison and it should fix the problem Kirill's highlighted.

Reply:
Fixed